draft-song-dns-wireformat-http-01.txt

```diff
@@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
 
    Proxy-DNS-Transport: xyz
```

Comment:
I think it should be mandatory.

On March 9, 2016 10:19:56 PM PST, "Linjian Song (Davey)" <notifications@github.com> wrote:
>> @@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over
>HTTP          February 2016
>>  
>>     Proxy-DNS-Transport: xyz
>
>should we make this header field optional or mandatory? if mandatory we
>should ask server always to check and follow this field. if not
>optional, then the server should have a default behavior if this field
>missing.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/songlinjian/DNS_HTTP/pull/6/files#r55638254

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.